Dockerfiles/Dockerfile.build

```diff
@@ -0,0 +1,38 @@
+FROM public.ecr.aws/debian/debian:buster-slim
```

Comment:
We use AL2 (and Alpine Linux) everywhere else, so why use debian here?

Reply:
I chose debian:buster-slim because it's a small image, and I'm installing all the packages required.